rules/library.bzl

```diff
@@ -552,6 +552,7 @@ def apple_library(
     # * Optionally, allow the consumers to set generate_default_umbrella_header to False, so the
     #   generated umbrella header does not contain any imports
     generate_default_umbrella_header = kwargs.pop("generate_default_umbrella_header", True)
+    generate_default_modulemap = kwargs.pop("generate_default_modulemap", True)
```

Comment:
Does the existing hook for being able to override the `modulemap_generator` enable this functionality in your `apple_library` wrapper? It'll still pass through kwargs, so if you need this set on a per-target basis, that should be possible.

Reply:
We do set `generate_default_umbrella_header` to `False` already, but since it does:

```bzl
generate_default_umbrella_header = kwargs.pop("generate_default_umbrella_header", True)
```

The kwargs don't reach the custom `modulemap_generator`. And if we pass a custom argument to the library target, rules_ios would attempt to forward to one of the libraries which fails too